components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -1070,6 +1088,35 @@ protected void actOnOneGroup(Set<Long> pixelsIds) {
         return value;
     }
 
+    /*
+     * (non-Javadoc)
+     *
+     * @see ome.api.ThumbnailStore#getThumbnail(ome.model.core.Pixels,
```

Comment:
This is failing javadoc: https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/ICE=3.6,jdk=8_LATEST,label=octopus/1064/warnings42Result/package.915837647/source.6931003519486980496/#1250